### ISSUE TITLE
Problem generating ISPyB XML from Eiger 4M data

### ISSUE THREAD

**Issue by olofsvensson** [xia2/xia2#109]
This is the stack trace:

```
opid30@mxhpc2-1705:/data/id30a3/inhouse/opid30a3/20170316/PROCESSED_DATA/trypsin_helical2/autoprocessing_trypsin_helical2_run1_1/XIA2_DIALS/mxoar3-2304_20170316-163746/Xia2DIALSv1_0_anom$ xia2.ispyb_xml 
Traceback (most recent call last):
  File "/opt/pxsoft/ccp4/v7.0/linux-x86_64/ccp4-7.0/lib/python2.7/runpy.py", line 162, in _run_module_as_main
    "__main__", fname, loader, pkg_name)
  File "/opt/pxsoft/ccp4/v7.0/linux-x86_64/ccp4-7.0/lib/python2.7/runpy.py", line 72, in _run_code
    exec code in run_globals
  File "/mntdirect/_sware/exp/pxsoft/ccp4/v7.0/linux-x86_64/ccp4-7.0/lib/py2/site-packages/xia2/command_line/ispyb_xml.py", line 25, in <module>
    ispyb_xml('ispyb.xml')
  File "/mntdirect/_sware/exp/pxsoft/ccp4/v7.0/linux-x86_64/ccp4-7.0/lib/py2/site-packages/xia2/command_line/ispyb_xml.py", line 18, in ispyb_xml
    ISPyBXmlHandler.write_xml(xml_out, command_line)
  File "/opt/pxsoft/ccp4/v7.0/linux-x86_64/ccp4-7.0/lib/py2/site-packages/xia2/Interfaces/ISPyB/ISPyBXmlHandler.py", line 195, in write_xml
    image_name = sweep.get_all_image_names()[0]
  File "/opt/pxsoft/ccp4/v7.0/linux-x86_64/ccp4-7.0/lib/py2/site-packages/xia2/Schema/XSweep.py", line 445, in get_all_image_names
    for i in range(0, array_range[1]-array_range[0])]
  File "/opt/pxsoft/ccp4/v7.0/linux-x86_64/ccp4-7.0/lib/py2/site-packages/cctbx_project/dxtbx/imageset.py", line 744, in get_path
    return self.reader().get_path(self._image_index(index))
  File "/opt/pxsoft/ccp4/v7.0/linux-x86_64/ccp4-7.0/lib/py2/site-packages/cctbx_project/dxtbx/imageset.py", line 764, in _image_index
    raise IndexError('Index out of range')
IndexError: Index out of range
```

**Comment by graeme-winter** [comment]
Now testing

**Comment by graeme-winter** [comment]
Reproduced with latest revision

```
[gw56@cs04r-sc-com12-07 x110]$ xia2.ispyb_xml
Traceback (most recent call last):
  File "/dls/science/users/gw56/svn/cctbx/build/../modules/xia2/command_line/ispyb_xml.py", line 26, in <module>
    ispyb_xml('ispyb.xml')
  File "/dls/science/users/gw56/svn/cctbx/build/../modules/xia2/command_line/ispyb_xml.py", line 19, in ispyb_xml
    ISPyBXmlHandler.write_xml(xml_out, command_line)
  File "/dls/science/users/gw56/svn/cctbx/modules/xia2/Interfaces/ISPyB/ISPyBXmlHandler.py", line 194, in write_xml
    image_name = sweep.get_all_image_names()[0]
  File "/dls/science/users/gw56/svn/cctbx/modules/xia2/Schema/XSweep.py", line 445, in get_all_image_names
    for i in range(0, array_range[1]-array_range[0])]
  File "/dls/science/users/gw56/svn/cctbx/modules/cctbx_project/dxtbx/imageset.py", line 754, in get_path
    return self.reader().get_path(self._image_index(index))
  File "/dls/science/users/gw56/svn/cctbx/modules/cctbx_project/dxtbx/imageset.py", line 774, in _image_index
    raise IndexError('Index out of range')
IndexError: Index out of range```

Probably something we should fix before 1.5.0 dials release
